dts/riscv32/riscv32-fe310.dtsi

```diff
@@ -0,0 +1,175 @@
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+	compatible = "SiFive,FE310G-0002-Z0-dev", "fe310-dev", "sifive-dev";
+	model = "SiFive,FE310G-0002-Z0";
+	L27: cpus {
```

Comment:
I'll update them to be meaningful. One reason I didn't is that I want to make sure that the labels aren't used for generating anything, since we'd like to be able to do out-of-tree ports to Zephyr for new chips without editing the labels.